### PR TITLE
Add support for Swift 2.3 in Xcode 8

### PR DESCRIPTION
**Pull request by gfontenot**

### CODE REVIEW

File: Runes.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -471,9 +471,11 @@
 					};
 					F8624C441A645C9500C883B3 = {
 						CreatedOnToolsVersion = 6.1.1;
+						LastSwiftMigration = 0800;
```
Comment (ikesyo) [inline Runes.xcodeproj/project.pbxproj:479]: This should be added to all targets in the project.

Comment (gfontenot) [inline Runes.xcodeproj/project.pbxproj:479]: Huh, weird, I wonder why it wasn't added automatically. I'll add this manually.

File: Runes.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -944,6 +951,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "com.thoughtbot.$(PRODUCT_NAME:rfc1034identifier)";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SDKROOT = macosx;
+				SWIFT_VERSION = 2.3;
```
Comment (gfontenot) [inline Runes.xcodeproj/project.pbxproj]: Should this be added to all of the targets (including the tests)? It looks like it's only being added to the frameworks.

Comment (ikesyo) [inline Runes.xcodeproj/project.pbxproj]: I prefer setting this at the project level only.

Comment (gfontenot) [inline Runes.xcodeproj/project.pbxproj]: I'm going to set this at the project level to make sure.

**Comment by gfontenot** [comment]
Well the auto-migrator for this sucks, but I think doing it manually worked.

**Pull request merged**